README.md

```diff
@@ -31,7 +32,11 @@ We split this repo out into different sections, each one has a summary of what's
 
 ### Product & Engineering Teams
 
-Artsy product engineering is organized in product teams. Each team has a _purpose_, eg. _"Help galleries get the most out of Artsy and run their business better"_, and Key Performance Indicators, or KPIs, eg. _"The number of artworks uploaded by partners."_ Each team has members with different responsibilities, including a _Project Manager_, a _Designer_, a _Technical Lead_ and _Engineers_, depending on size. Sometimes the same person wears multiple hats.
+Artsy product engineering is organized in product teams. Each team has a _purpose_, eg. _"Help galleries get the
+most out of Artsy and run their business better"_, and Key Performance Indicators, or KPIs, eg. _"The number of
+artworks uploaded by partners."_ Each team has members with different responsibilities, including a _Project
+Manager_, a _Designer_, a _Technical Lead_ and _Engineers_, depending on size. Sometimes the same person wears
```

Comment:
Might we want to use the term Product Manager here instead of Project Manager?

Reply:
(good catch, updated!)